src/codegen/templates/property_deleter_template.py.mako

```diff
@@ -0,0 +1,37 @@
+<%def name="script_property_deleter(attribute)">\
+<%
+        from codegen.utilities.text_wrappers import wrap
+    %>\
+    @${attribute.name}.deleter
+    def ${attribute.name}(self):
+<%
+        argtypes = []
+        for handle_parameter in attribute.handle_parameters:
+            if handle_parameter.ctypes_data_type == 'ctypes.c_char_p':
+                argtypes.append('ctypes_byte_str')
+            else:
+                argtypes.append(handle_parameter.ctypes_data_type)
+    %>\
+    %if len(attribute.c_function_name) < 33:
+        cfunc = lib_importer.${'windll' if attribute.calling_convention=='StdCall' else 'cdll'}.DAQmxReset${attribute.c_function_name}
+    %else:
+        cfunc = (lib_importer.${'windll' if attribute.calling_convention=='StdCall' else 'cdll'}.
+                 DAQmxReset${attribute.c_function_name})
+    %endif
+        if cfunc.argtypes is None:
+            with cfunc.arglock:
+                if cfunc.argtypes is None:
+                    cfunc.argtypes = [
+                        ${', '.join(argtypes) | wrap(24, 24)}]
```

Comment:
LOVE these `wrap` and `docstring_wrap` helpers. I wish the parameters were more documenting, so I could understand what they mean when looking at a template. Can you make them named parameters here?

```python
wrap(init_indent=24, subsequent_indent=24)
```

Reply:
Actually, I gave a recommendation down below about how to eliminate `subsequent_indent` in these calls if they're equal. In that case a single parameter to wrap doesn't need to be named. It is very self-documenting!